zk/stages/stages.go

```diff
@@ -104,7 +104,12 @@ func SequencerZkStages(
 			ID:          stages2.Execution,
 			Description: "Sequence transactions",
 			Forward: func(firstCycle bool, badBlockUnwind bool, s *stages.StageState, u stages.Unwinder, tx kv.RwTx, quiet bool) error {
-				return SpawnSequencingStage(s, u, ctx, exec, history, quiet)
+				sequencerErr := SpawnSequencingStage(s, u, ctx, exec, history, quiet)
+				if sequencerErr != nil || u.IsUnwindSet() {
+					exec.legacyVerifier.CancelAllRequests()
+					// on the begining of next iteration the EXECUTION will be aligned to DS
```

Comment:
You should enable the DS <-> Sequencer alignment on next stage loop like [here](https://github.com/0xPolygonHermez/cdk-erigon/blob/zkevm/zk/stages/stages.go#L111). This is specific to zkevm/RC1 and it was not needed for beta branches.

Reply:
good find!  Not sure how the comment got there but the code didn't!  I have pushed the change to resolve this one.